parm/config/gfs/config.resources

```diff
@@ -400,6 +401,34 @@ case ${step} in
     export memory_ocnanalrun
     ;;
 
+  "ocnanalecen")
+    npes=16
+    case ${CASE} in
+      "C384")
+        npes=40
+        memory_ocnanalecen="96GB"
+        ;;
+      "C96")
+        npes=16
+        memory_ocnanalecen="96GB"
+        ;;
+      "C48")
+        npes=16
+        memory_ocnanalecen="24GB"
+        ;;
+      *)
+        echo "FATAL ERROR: Resources not defined for job ${job} at resolution ${CASE}"
+        exit 4
+    esac
```

Comment:
Is the ocean recentering job dependent on atmosphere resolution?
Probably should have caught this on the other `ocn` jobs as well.

Reply:
Ha! Good point. No it isn't, we didn't upgrade the scripting after the ocean resolution was added as an option.

Reply:
What should it be?

Reply:
> What should it be? The entries for the ocean all key off of the atmospheric resolution.

@AndrewEichmann-NOAA , the variable is `OCNRES`, and I think the cases are `0.25`, `1.00` and `5.00`. That or x100, you'll have to check ...

Reply:
Ok should be fixed @guillaumevernieres @aerorahul

Reply:
looks like you've got the resource allocation for the resolutions backward @AndrewEichmann-NOAA .

Reply:
@guillaumevernieres d'oh